frontend/src/features/properties/list/PropertyListView.tsx

```diff
@@ -93,10 +93,10 @@ const defaultFilterValues: IPropertyFilter = {
 };
 
 export const flattenProperty = (apiProperty: IApiProperty): IProperty => {
-  const assessedLand = getMostRecentEvaluation(apiProperty.evaluations, EvaluationKeys.Assessed);
+  const assessedLand = getCurrentYearEvaluation(apiProperty.evaluations, EvaluationKeys.Assessed);
```

Comment:
Correct me if I'm wrong, but I believe that we consistently only want to display current year values for assessments on the list view as well as the project views.

Reply:
correct